fast_response/FastResponseAnalysis.py

```diff
@@ -992,6 +1020,424 @@ def unblind_TS(self, custom_events=None):
             return ts, ns, gamma_fit
         else:
             return ts, ns
+    def inject_events(self,ninj=0):
+        ni, sample,src_ra, src_dec = self.inj.sample(ninj,poisson=False,return_position =True)
+        return sample
+
+
+    def make_prior_map(self, 
+                        plotting_location=None,
+                        format=True
+                           ):
+        """
+        Generates a zoomed in skymap around the best fit location from the maximum likelihood analysis        
+        """
+        #Format the skymap if told to format (usually formatted at construction so shouldn't change anything)
+        if(format):
+            skymap = self.format_skymap(self.skymap)
+        else:
+            skymap=self.skymap
+        
+        #Plot the prior skymap
+        fig, ax=plt.subplots(1,1,figsize=(10,10))
+        plt.sca(ax)
+        reso=.1
+        sizeZoomInDegs=10
+        nPixZoom=sizeZoomInDegs/reso*60
+        hp.visufunc.gnomview(map=np.log10(skymap),hold=True,title="Prior Skymap",rot=(np.degrees(self.skymap_fit_ra),np.degrees(self.skymap_fit_dec),0),
+            xsize=nPixZoom,ysize=nPixZoom,reso=reso,bgcolor="white",badcolor="white",margins=(.02,.02,.05,0),notext=True,cbar=False)
+            
+        hp.graticule(verbose=True)
+        plotting_utils.plot_labels(self.skymap_fit_dec,self.skymap_fit_ra, reso,nPix=nPixZoom,label_scale=.97)
+        
+        ax=plt.gca()
+        text_objs = ax.findobj(
+                lambda artist: isinstance(artist, Text)
+            )
+        for obj in text_objs:
+            obj.set_fontsize(30)
+        plotting_utils.plot_color_bar(range=[0,np.nanmax(self.skymap)], cmap="viridis", col_label=r"Skymap Prior PDF",
```

Comment:
can you add more spacing here to make this easier to read? a new blank line after this function, and make some of these args their own line for readability